Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1381,12 +1377,110 @@
+    <!-- Korigiranje crte - kad je u sredini prvog reda -->
+    <RegEx find="([.!?&quot;&gt;]) - ([A-ZČĐŠŽčš'&quot;&lt;])" replaceWith="$1 -$2" />
+    <!-- Zatvoren tag pa razmak poslije crtice -->
+    <RegEx find="(&gt;) - ([A-ZČĐŠŽčš„'&quot;])" replaceWith="$1 -$2" />
+    <!-- Zatvoren tag pa crtica razmak -->
+    <RegEx find="(&gt;)- ([A-ZČĐŠŽčš„'&quot;])" replaceWith="$1-$2" />
+    <!-- Zagrada pa crtica razmak -->
+    <RegEx find="\(- ([A-ZČĐŠŽčš„'&quot;])" replaceWith="(-$1" />
+    <!-- Smart space after dot -->
+    <!-- osim kad je zadnje t (riječ kolt) -->
+    <RegEx find="([a-su-zá-úñä-ü])\.([^\s\n().:?!*^“”'&quot;&lt;])" replaceWith="$1. $2" />
+    <!-- Oznaka za kalibar. Npr. "Colt .45" -->
+    <!-- Da bi radilo, da bi ovaj razmak bio dozvoljen, u odčekirajte "Razmaci ispred točke" -->
+    <RegEx find="t\.([0-9][0-9])" replaceWith="t .$1" />
+    <!-- Joey(j)a -->
+    <RegEx find="([A-Z][a-z])eyj([a-z])" replaceWith="$1ey$2" />
+    <!-- Sređuje zarez sa razmakom -->
+    <RegEx find="([A-ZČĐŠŽa-zčđšžá-úñä-ü&quot;]),([^\s\n(),?!“&lt;])" replaceWith="$1, $2" />
+    <RegEx find=" , " replaceWith=", " />
+    <RegEx find=" ,([A-ZČĐŠŽa-zčđšž])" replaceWith=", $1" />
+    <RegEx find=" ,$" replaceWith="," />
+    <RegEx find="([?!])-" replaceWith="$1 -" />
+    <!-- Space after last of some consecutive dots (eg. "...") -->
+    <RegEx find="([a-zčđšž])(\.{3}|!)([a-zčđšž])" replaceWith="$1$2 $3" />
+    <!-- Delete space after "..." that is at the beginning of the line. You may delete this line if you don't like it -->
+    <!-- <RegEx find="^\.{3} +" replaceWith="..." /> -->
+    <!-- "tekst ... tekst" mijenja u "tekst... tekst" -->
+    <RegEx find="([A-ZČĐŠŽa-zčđšž]) \.{3} " replaceWith="$1... " />
+    <RegEx find="(\S)\. &quot;" replaceWith="$1.&quot;" />
+    <RegEx find="&quot; \." replaceWith="&quot;." />
+    <RegEx find="(\S\.{3}) &quot;(\s|$)" replaceWith="$1&quot;$2" />
+    <RegEx find=" \.{3}$" replaceWith="..." />
+    <RegEx find="([a-zčđšž])( \.{3}|\.{2}$)" replaceWith="$1..." />
+    <!-- Razmak ispred zagrade -->
+    <RegEx find="([A-ZČĐŠŽa-zčđšž])\(" replaceWith="$1 (" />
+    <!-- Razmak iza upitnika -->
+    <RegEx find="\?([A-ZČĐŠŽčš])" replaceWith="? $1" />
+    <RegEx find="(^|&gt;)\.{3} ([A-ZČĐŠŽčš])" replaceWith="$1...$2" />
+    <!-- Brise ... kad je na poc. reda "... -->
+    <RegEx find="^&quot;\.{3} " replaceWith="&quot;" />
+    <RegEx find="([0-9])\$" replaceWith="$1 $" />
+    <!-- ti š -> t š by Strider -->
+    <!-- Zamijeni sva "**ti šu*" s "**t šu*" i "**ti še*" s "**t še*" -->
+    <!-- <RegEx find="([a-z])ti (š+[eu])" replaceWith="$1t $2" /> -->
+    <!-- <RegEx find="([A-Za-z])ti( |\r?\n)(š[eu])" replaceWith="$1t$2$3" /> -->
+    <!-- <RegEx find="(?i)\b(ni)t (š[eu])" replaceWith="$1ti $2" /> -->
+    <!-- Razmak poslije <i> i poslije .. -->
+    <RegEx find="^(&lt;[i|b|u]&gt;) +" replaceWith="$1" />
```

Comment:
Here is some changes I have done https://github.com/ivandrofly/subtitleedit/commit/c12171f68b6291d6104307a9772d32992300463b